packages/webhooks/schema/models/WebhookSubscription.test.js

```diff
@@ -253,49 +254,120 @@ const WebhookSubscriptionBasicTests = (appName, actorsInitializer) => {
-                expect(subscription).toHaveProperty('failuresCount', 0)
-                const [failuresUpdate] = await updateTestWebhookSubscription(actors.admin, subscription.id, {
-                    failuresCount: 1,
+            describe('"failuresCount" field', () => {
+                const cases = [
+                    ['syncedAt', dayjs().toISOString()],
+                    ['syncedAmount', 30],
+                    ['model', 'User'],
+                    ['fields', '{ id }'],
+                    ['filters', {}],
+                    ['maxPackSize', 10],
+                    ['operation', 'update'],
+                ]
+                test.each(cases)('"failuresCount" should be reset on "%s" field update', async (fieldName, fieldValue) => {
+                    const [subscription] = await createTestWebhookSubscription(actors.admin, hook)
+                    expect(subscription).toBeDefined()
+                    expect(subscription).toHaveProperty('failuresCount', 0)
+                    const [onlyAmountUpdate] = await updateTestWebhookSubscription(actors.admin, subscription.id, {
+                        failuresCount: 1,
+                    })
+                    expect(onlyAmountUpdate).toHaveProperty('failuresCount', 1)
+                    const [onlyTimeUpdate] = await updateTestWebhookSubscription(actors.admin, subscription.id, {
+                        [fieldName]: fieldValue,
+                    })
+                    expect(onlyTimeUpdate).toHaveProperty('failuresCount', 0)
+                    const [bothUpdate] = await updateTestWebhookSubscription(actors.admin, subscription.id, {
+                        [fieldName]: fieldValue,
+                        failuresCount: 1,
+                    })
+                    expect(bothUpdate).toHaveProperty('failuresCount', 1)
+                    await softDeleteTestWebhookSubscription(actors.admin, subscription.id)
                 })
-                expect(failuresUpdate).toHaveProperty('failuresCount', 1)
-                const [batchSentUpdate] = await updateTestWebhookSubscription(actors.admin, subscription.id, {
-                    syncedAmount: 30,
+            })
+            describe('"operations" field', () => {
+                describe('Must throw validation error on wrong operations', () => {
+                    const cases2 = [
+                        ['String', 'some string'],
+                        ['Number', 213],
+                        ['Boolean', true],
+                        ['Object', { update: true }],
+                        ['Array', ['update']],
+                    ]
+                    test.each(cases2)('%s', async (name, operation) => {
+                        await expectToThrowGraphQLRequestError(async () => {
+                            await createTestWebhookSubscription(actors.admin, hook, {
+                                operation,
+                            })
+                        }, 'got invalid value')
```

Comment:
Include field name here, since it's not 100% that error is thrown by `operation` field